JSDemos/Demos/TreeList/UsingSearchPanel/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to&nbsp;filter data by&nbsp;all columns, that&nbsp;is, to&nbsp;perform search. Type a&nbsp;search string into the search panel located in&nbsp;the top-right corner. The TreeList tries to&nbsp;find entries of&nbsp;your search string in&nbsp;the rows. If&nbsp;such entries are found, the TreeList displays the full hierarchy down to&nbsp;matching rows, but not further.
\ No newline at end of file
+If you want users to search data, display the [searchPanel](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/searchPanel/). Set its [visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/searchPanel/#visible) property to **true**.
+
+The TreeList searches in all columns, regardless of whether they are [visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#visible) or hidden. To exclude hidden columns from search, enable the **searchPanel**.[searchVisibleColumnsOnly](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/searchPanel/#searchVisibleColumnsOnly) property. You can also exclude any specific column. To do so, set its [allowSearch](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowSearch) property to **false**
+
+Search is case-insensitive. Numeric, date, and Boolean values match only if they are equal to the search query. String values match even if they contain only a part of the query.
```

Comment:
1. I'd have the sentences about "case insensitive" and "string values" together, not separated. Most likely, I'd move the "case-insensitive" sentence to the end. It doesn't apply to numeric/date-time/Boolean values anyway, right? 
2. If I'm looking for "12" and a numeric column contains the number 123, it's not a match? That's what the sentence says, I'm just making sure. Or I can imagine that I want to find all dates in December and my search string is "12/". But as I understand, the search will not find any date-time values if I specify such a string? 
3. The last sentence - I'm also not sure about it. What you say is that I can enter "Vladimir". A part of the query is "Vlad". If a cell value contains the string "Vladislav", then it does contain "a part of the query", which is "Vlad". So I enter Vladimir and Vladislav is a match? Sounds strange.

Reply:
1) "case-insensitive" is only for strings. Will move this sentence.
2) Right, search will not find any date-time values for such strings. They should be equal to, not contain the search query.
3) I mean that the query "ab" will list "abcd" and "abzy" values. "String values match even if they are partially equal to the search query."?

Reply:
1. Thanks
2. Alright - I was just making sure. But then the following question arises. Obviously dates or numbers can be formatted differently based on locale or maybe column settings. It could be 1 or $1.00. Or it could be 12/31/2020 or 31.12.2020. You say "values match" ... and I'm not sure which of those options are "values". The cell text needs to match the query or actually some underlying value? And if I show check boxes for the Boolean column, can I still search for "true"? Or "True"? 
3. OK - that's clearer, thanks. I'd probably say "contains". "String values match if they contain the search query".

Reply:
2) If column's type is "currency" (e.g. the displayed values is "$2"), then the queries "$2" and "2" will list the corresponding row.
The search for dates works when the date is full (12/31/2020). It also works when a column uses a custom format ({ type: "longDate" }), but the query should be "Sunday, October 10, 2020". For custom scenarios, developers have to implement calculateFilterExpression. 
If column's type is "boolean", rows will be shown when the query is "true" (not "True"). We've decided to leave it as it is.